app/models/container_group.rb

```diff
@@ -69,6 +71,14 @@ def event_where_clause(assoc = :ems_events)
     end
   end
 
+  def tenant_identity
+    if ext_management_system
+      ext_management_system.tenant_identity
+    else
+      User.super_admin.tap { |u| u.current_group = Tenant.root_tenant.default_miq_group }
+    end
```

Comment:
Not for this pr but @jrafanie this method seems to repeat multiple times (5 times before this patch) should we extract it to somewhere? can you think of a good place for it?